benchmark/blockmap.ts

```diff
@@ -0,0 +1,78 @@
+import { createReadStream, readFileSync } from 'fs';
```

Comment:
When I joined, Hedley explained that we should add the licence boilerplate at the top of every file we create or modify. If so, a conversion to typescript seems like a good opportunity to do it:
```
/**
 * @license
 * Copyright 2019 Balena Ltd.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *    http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
```

Reply:
And the CI complains about it apparently, so it's certainly something we should do

Reply:
added